lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -289,8 +291,10 @@ def error_code_from(succeeded, response)
         unless succeeded
           if errors = response["errors"]
             errors.first.try(:[], "code")
+          elsif status = response["statusOutput"]["statusCode"]
```

Comment:
Wondering if we should use `response.try(:[], "statusOutput").try(:[] "statusCode")`? The Global Collect API responses seem to have a number of different structures so in the chance we get a response that doesn't have either types we won't run into an exception trying to reference `statusCode` of `nil`.